tests/installation/bootloader_zkvm.pm

```diff
@@ -32,10 +32,7 @@ sub set_svirt_domain_elements {
         my $ntlm_p = get_var('NTLM_AUTH_INSTALL') ? $ntlm_auth::ntlm_proxy : '';
         my $cmdline = get_var('VIRSH_CMDLINE') . $ntlm_p . " ";
         if (is_agama) {
-            $cmdline .= " root=live:http://" . get_var('OPENQA_HOSTNAME') .
-              ((get_var('FLAVOR') eq "Full") ?
-                  "/assets/repo/" . get_required_var('REPO_0') . "/LiveOS/squashfs.img" :
-                  "/assets/iso/" . get_required_var('ISO'));
+            $cmdline .= " root=live:http://" . get_var('OPENQA_HOSTNAME') . "/assets/repo/" . get_required_var('REPO_0') . "/LiveOS/squashfs.img";
```

Comment:
I think it is right time, to use `MIRROR_HTTP`  for `zkvm` and `pvm` only in the same way like it is for `ipxe`  and get rid of` OPENQA_HOSTNAME` and `REPO_0` at least for Agama based installation.